server/constants/tours/eyeSpy/apr-2022.json

```diff
@@ -0,0 +1,92 @@
+{
+  "title": "Eye Spy Gallery Scavenger Hunt",
+  "subtitle": "",
+  "heroImageInvno": "BF138",
+  "heroImageStyle": {
+    "overlay": {
+      "backgroundColor": "rgba(0, 0, 0, 0.3)"
+    }
+  },
+  "description": "<p>Can you spy all of these works in our galleries? Find a member of the Barnes team wearing a white apron to mark off your sheet when you find them and turn in the completed sheet for a prize!</p>",
+  "body": {
+    "from": 0,
+    "size": 25,
+    "_source": [
+      "id",
+      "title",
+      "people",
+      "medium",
+      "imageOriginalSecret",
+      "imageSecret",
+      "ensembleIndex",
+      "objRightsTypeId",
+      "onview",
+      "invno",
+      "image",
+      "curatorialApproval",
+      "shortDescription",
+      "nationality",
+      "birthDate",
+      "deathDate",
+      "artistPrefix",
+      "artistSuffix",
+      "culture",
+      "displayDate",
+      "medium",
+      "dimensions",
+      "creditLine",
+      "longDescription",
+      "bibliography",
+      "exhHistory",
+      "publishedProvenance"
+    ],
+    "query": {
+      "bool": {
+        "filter": {
+          "terms": {
+            "invno": [
+              "01.02.44",
+              "bf831",
+              "01.06.16",
+              "01.08.01",
+              "bf138",
+              "bf2",
+              "01.18.10",
+              "bf2512",
+              "01.20.21"
+            ]
```

Comment:
I didn't notice until after it was merged, but these inventory numbers need to match the ones that should be included in the tour.